src/framework/auth/services/interceptors/jwt-interceptor.ts

```diff
@@ -13,24 +14,41 @@ export class NbAuthJWTInterceptor implements HttpInterceptor {
   }
```

Comment:
so we can pass the public urls here as an injectable token for this particular interceptor.